### ISSUE TITLE
Request: Sky Bet

### ISSUE THREAD

**Issue by AlexeiBlue** [simple-icons/simple-icons#10479]
### Brand Name

Sky Bet

### Website

https://m.skybet.com/

### Popularity Metric

The similarweb rank is: 3,039. See https://www.similarweb.com/website/skybet.com

### Official Resources for Icon and Color

Skybet https://bet.sbgcdn.com/static/mbet/img/content/logos/skybet-rebrand.png

### Additional Comments

The favicon is also used occasionally: https://bet.sbgcdn.com/static/mbet/img/favicon.png

**Comment by PeterShaggyNoble** [comment]
Thanks for the request, @AlexeiBlue.

For anyone looking to take this on: there's [an SVG](https://static.skybetservices.com/core-static-assets/skybet-alt-v2.svg) on their [registration page](https://www.skybet.com/secure/identity/m/register/step-1/mskybet).

**Comment by AlexeiBlue** [comment]
Amazing find, I didn't realise that existed :)

**Comment by PeterShaggyNoble** [comment]
> Amazing find

It's what we do! 😁

As @performant23 has seemingly abandoned their attempt at it and assuming they're not still working on it, would you like to have a read through our contributing guidelines and give this a try yourself?

**Comment by performant23** [comment]
Yeah, @PeterShaggyNoble. I found that vectorizing is a bit complex for this logo. So, yeah, all the best for anyone working on this!

**Comment by AlexeiBlue** [comment]
Will give it ago, might take me some time though :D

**Comment by AlexeiBlue** [comment]
@PeterShaggyNoble Hopefully the attached PR looks okay?  I had a good read through the contributor file but let me know if I've missed anything, hopefully now I've got used to the process I can do the same for all our other brands and just submit the issues with PR's ready to go

**Comment by service-paradis** [comment]
According to section "24. Intellectual Property" from https://support.skybet.com/s/article/Sky-Betting-and-Gaming-General-Terms-Conditions, I'm not sure we can add this one:

> 24.2 The names, images and logos identifying us, our partners or third parties and our/their products and services contained in our websites and mobile and tablet applications are proprietary marks and may not be reproduced or otherwise used without express permission.

**Comment by AlexeiBlue** [comment]
> According to section "24. Intellectual Property" from https://support.skybet.com/s/article/Sky-Betting-and-Gaming-General-Terms-Conditions, I'm not sure we can add this one without permission:
> 
> > 24.2 The names, images and logos identifying us, our partners or third parties and our/their products and services contained in our websites and mobile and tablet applications are proprietary marks and may not be reproduced or otherwise used without express permission.

I've emailed our legal team to ask about this, hopefully they can give some guidance on what to do next 🤞

**Comment by PeterShaggyNoble** [comment]
Have you heard anything back from them, @AlexeiBlue? Do you work for them?

**Comment by AlexeiBlue** [comment]
Hey @PeterShaggyNoble sorry for the long delay, I do work for Sky Betting and Gaming which belongs to Flutter.  Unfortunately I never heard back from Legal, I've sent them another email to chase it up

**Comment by PeterShaggyNoble** [comment]
Any update from your legal team, @AlexeiBlue?

**Comment by adamrusted** [comment]
Hey @AlexeiBlue - as per updated guidelines, we can only give the legal team until 3rd July to reply before we have to close the PR. Can always reopen if we _do_ hear back from them.

**Comment by AlexeiBlue** [comment]
Hey @PeterShaggyNoble and @adamrusted apologies for the delay, unfortunately I haven't had a response from our legal team despite multiple attempts.  I'll try again and see what happens but I suspect this PR will get closed off unfortunately

**Comment by AlexeiBlue** [comment]
Had a word with Legal today :D they think this will be perfectly fine but they're gonna get back to me by the end of the week

**Comment by adamrusted** [comment]
I'm assuming they didn't get back to you by the end of the week @AlexeiBlue? ;)

**Comment by AlexeiBlue** [comment]
Hey @adamrusted unfortunately I didn't, I've sent a follow up message but I believe the person I spoke to in legal is on holiday.  Looks like we're past the deadline for this request so if needs be feel free to close it for now and I'll re-raise it if legal get back to me :)
